More details on GDAL create options (#3755)
qgis/QGIS-Documentation: source/docs/user_manual/managing_data_source/create_layers.rst

Raster specific parameters
--------------------------

> Depending on the format of export, some of these options are available or not:

Reviewer: ```suggestion
Depending on the format of export, these options may or may not be available:
```

* :guilabel:`Output mode` (it can be **raw data** or **rendered image**)
* :guilabel:`Format`: exports to any raster format GDAL can write to, such as
  GeoTiff, GeoPackage, MBTiles, Geospatial PDF, SAGA GIS Binary Grid,
  Intergraph Raster, ESRI .hdr Labelled...
* :guilabel:`Resolution`

> * :guilabel:`Create Options`: allows to use the advanced options (file compression,

Reviewer: ```suggestion
* :guilabel:`Create Options`: use advanced options (file compression,
```
